src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -50,6 +50,12 @@ export type ColorPickerProps = {
    * @default false
    */
   setFocus?: boolean;
+  /**
+   * If true, then alpha value is shown so it can be set,
+   * opacity slider will also be visible.
```

Comment:
This comment could be improved. Something like:

```
If true, ColorBuilder will show the alpha slider and ColorInputPanel will show an alpha input.
```

Reply:
Updated comment